matrix-sdk/src/main/java/org/matrix/androidsdk/features/integrationmanager/IntegrationManager.kt

```diff
@@ -68,8 +56,8 @@ class IntegrationManager(val mxSession: MXSession, val context: Context) {
     var integrationAllowed = true
         private set
 
-    private var widgetPermissions = emptyList<WidgetPermission>()
-    private var videoConfDomainPermissions = emptyList<DomainPermission>()
+    private var widgetPermissions = emptyMap<String, Boolean>()
+    private var nativeWidgetPermissions = emptyMap<String, Map<String, Boolean>>()
```

Comment:
Maybe document what are the keys and what are the values please

Reply:
done